### DISCUSSION TITLE
Best model for advance machine translation?

### DISCUSSION THREAD

**Discussion by michalss** [betalgo/openai#178] [Q&A]
HI,

Im looking for best way to use GPT for text translation. Can you please advice me how, or what enpoint use it for that ? I would like to also pass instruction for context and glossary.

thank you for help

**Comment by Frogley** [comment]
According to the official documentation, it is recommended to use gpt-3.5-turbo for most use cases, as it performs at a similar capability to text-davinci-003 but at 10% the price per token. For example, if you need translate English to French, you can use following conversation:
```json
[
  {"role": "user", "content": 'Translate the following English text to French: "{text}"'}
]
```
